tests/testStreamOpen.cpp

```diff
@@ -20,6 +20,15 @@
 
 using namespace oboe;
 
+class NumFramesCallback : public AudioStreamCallback {
```

Comment:
I feel like `NumFramesCallback` doesn't make it obvious what this class is doing. Could it be renamed to something like `CallbackSizeMonitor`

Reply:
Done